func_tests/test_client.py

```diff
@@ -73,3 +75,70 @@ def test_batch_change(vinyldns_test_context):
     assert change2.record.ptrdname == 'change-test.vinyldns.'
     assert change2.type == RecordType.PTR
 
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[0].zone_id, r.changes[0].record_set_id)
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[1].zone_id, r.changes[1].record_set_id)
+
+
+def test_batch_change_review_process_reject(vinyldns_test_context):
+    changes = [
+        AddRecord('test-approve-success.not.loaded.', RecordType.A, 200, AData("4.3.2.1"))
+    ]
+
+    bc = vinyldns_test_context.client.create_batch_change(
+        BatchChangeRequest(changes, 'comments', vinyldns_test_context.group.id)
+    )
+
+    assert bc.status == 'PendingReview'
+    assert bc.approval_status == 'PendingReview'
+    assert len(bc.changes[0].validation_errors) == 1
+
+    vinyldns_test_context.support_client.reject_batch_change(bc.id, 'cannot create the zone')
+
+    completed_bc = vinyldns_test_context.client.get_batch_change(bc.id)
+
+    assert completed_bc.status == 'Rejected'
+    assert completed_bc.approval_status == 'ManuallyRejected'
+    assert completed_bc.reviewer_id == 'support-user-id'
+    assert completed_bc.reviewer_username == 'support-user'
```

Comment:
getting error `'BatchChange' object has no attribute 'reviewer_username'`, you have it as `reviewer_user_name`, I would probably stick with `reviewer_username`

Reply:
I did have it as `reviewer_username` everywhere and it worked in the func tests, but not in the unit tests or when i actually tested it locally.  I don't see where we're explicitly using the `camel_to_underscore` method in serdes but it seems like that or something similar is applied..